### DISCUSSION TITLE
Orderbook depth

### DISCUSSION THREAD

**Discussion by Cosmingld** [bmoscon/cryptofeed#860] [Q&A]
I am trying to get orderbook depth data from FTX and Binance.

As an example, I know that FTX has a max orderbook datafeed of 100. Does L2_BOOK of FTX when using influxdb automatically default to 100 depth? If not, how can I change the depth that cryptofeed streams?

If someone can reference a orderbook example with cryptofeed, i would appreciate it.

**Comment by bmoscon** [comment]
you can use the depth= kwarg to change the depth. it will default to the max usually.
